explorer/client/src/__tests__/components/JobRuns/TaskRuns.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { mount } from 'enzyme'
+import { v4 as uuid } from 'uuid'
+import React from 'react'
+import TaskRuns from '../../../components/JobRuns/TaskRuns'
+
+const etherscanHost = 'ropsten.etherscan.io'
+
+describe('components/JobRuns/TaskRuns', () => {
+  it('hides incoming pending confirmations with NO minimumConfirmations', () => {
+    const taskRuns = [
```

Comment:
Should it be taskRuns: Array<ITaskRun> ?

Reply:
That type will already be inferred by typescript.